### ISSUE TITLE
Tabs that are just links to external sites

### ISSUE THREAD

**Issue by JonEsparaz** [thewca/worldcubeassociation.org#8110]
**Is your feature request related to a problem? Please describe.**
If we want to link to an external site (e.g., for groups or a staff application form), we usually create an entire tab, then write something like "blah, blah, click here".

**Describe the solution you'd like**
Be able to provide a link instead of markdown for a tab, then when you click on the tab, it just brings you to the external site in a new browser tab.

**Describe alternatives you've considered**
Tools that are commonly used (e.g., WCA Live, arguably competitiongroups.com) can be optionally linked on the WCA page (i.e., we only allow these sites, not just random URLs).

I know that WCA Live sorta does this already when the competition is in progress with this banner, but it's not a link that is permanently on the page:

![image](https://github.com/thewca/worldcubeassociation.org/assets/48423418/d38b9cfa-3399-4892-8783-f8d67f44931e)

**Additional context**
N/A

**Comment by dunkOnIT** [comment]
We have used a [linktree](https://linktr.ee/cubingza) for CubingZA competitions to give people easy access to relevant competition links. Could an "Important Links" tab on the comp page fulfill this purpose? In the future this could perhaps become a component on the Info tab of the competition.

**Comment by JonEsparaz** [comment]
Yes, a Linktree or Linktree-like tab could work, but it's still two or more clicks.

(This is by no means high priority, just a nice-to-have.)
